includes/Modules/Analytics.php

```diff
@@ -1368,6 +1393,15 @@ protected function create_analytics_site_data_request( array $args = array() ) {
 		$request->setIncludeEmptyRows( true );
 		$request->setViewId( $profile_id );
 
+		$dimension_filter_clauses = array();
+		if ( ! empty( $args['dimension_filters'] ) ) {
+			$dimension_filters       = $args['dimension_filters'];
+			$dimension_filter_clause = new Google_Service_AnalyticsReporting_DimensionFilterClause();
+			$dimension_filter_clause->setFilters( array( $dimension_filters ) );
+			$dimension_filter_clause->setOperator( 'AND' );
+			$dimension_filter_clauses[] = $dimension_filter_clause;
+		}
+
```

Comment:
Nitpick, but maybe move this a bit further down to be right above the `if ( ! empty( $args['page'] ) ) {` clause? This way all logic touching dimension filter clauses is located close together.

Reply:
Updated.